darts/metrics/metrics.py

```diff
@@ -872,9 +933,101 @@ def r2_score(actual_series: Union[TimeSeries, Sequence[TimeSeries]],
     float
         The Coefficient of Determination :math:`R^2`
     """
-    y1, y2 = _get_values_or_raise(actual_series, pred_series, intersect)
+    y1, y2 = _get_values_or_raise(actual_series, pred_series, intersect, stoch_reduction)
     y1, y2 = _remove_nan_union(y1, y2)
     ss_errors = np.sum((y1 - y2) ** 2)
     y_hat = y1.mean()
     ss_tot = np.sum((y1 - y_hat) ** 2)
     return 1 - ss_errors / ss_tot
+
+
+@multi_ts_support
+@multivariate_support
+def mpgl(
+                       actual_series: Union[TimeSeries, Sequence[TimeSeries]],
+                       pred_series: Union[TimeSeries, Sequence[TimeSeries]],
+                       intersect: bool = True,
+                       *,
+                       deviation: float = 1.0,
+                       reduction: Callable[[np.ndarray], float] = np.mean,
+                       inter_reduction: Callable[[np.ndarray], Union[float, np.ndarray]] = lambda x: x,
+                       n_jobs: int = 1,
+                       verbose: bool = False
+):
+    """ MEAN PERCENTAGE GAUSSIAN LIKELIHOOD
+
+    Computes the mean gaussian likelihood of the ground truth series `actual_series` ± deviation,
```

Comment:
Actually if I understand it well, it computes the mean probability that the ground truth values fall within X +- deviation, where X is the mean of `pred_series`. A few things:
* You should describe it as mean probability (instead of mean likelihood)
* The meaning of `deviation` is a bit weird and left to the users. How about constraining `deviation` to represent a multiple of the standard deviation. This way, for instance for `deviation=2`, we could interpret this metric as "The average probability (over time) that the ground truth lie in the +- 2σ interval defined by the `pred_series`, assuming a Gaussian distribution". `deviation=2` could be a nice default as it corresponds to 95.4% probability.
* You could also optionally avoid using `deviation` (say, if it's None), and in this case compute the mean likelihood (instead of a mean probability). The likelihood has the advantage that it doesn't depend on a (somewhat arbitrary) integration region (given by `deviation`), but has the disadvantage that likelihood values are not bound between 0 and 1 and thus harder to interpret.